### ISSUE TITLE
kr3 crashing on linux / TooMuchFreeTime

### ISSUE THREAD

**Issue by cigumo** [cigumo/krli#406]
https://steamcommunity.com/app/816340/discussions/1/1734342161853069541/

```
Driver: Intel Open Source Technology Center Mesa DRI
Intel(R) HD Graphics 520 (Skylake GT2) x86/MMX/SSE2
Driver Version: 3.0 Mesa 12.0.6
OpenGL Version: 3.0
Desktop Color Depth: 24 bits per pixel
```

```
ERROR

./lib/klove/kui.lua:300: Shader uniform 'outline_color' does not exist.
A common error is to define but not use the variable.

Traceback

./main.lua:491: in function <./main.lua:449>
[C]: in function 'send'
./lib/klove/kui.lua:300: in function '_draw_self'
./lib/klove/kui.lua:720: in function 'draw'
./lib/klove/kui.lua:834: in function '_draw_children'
./lib/klove/kui.lua:727: in function 'draw'
./lib/klove/kui.lua:1586: in function 'draw'
./lib/klove/kui.lua:834: in function '_draw_children'
./lib/klove/kui.lua:727: in function 'draw'
./lib/klove/kui.lua:834: in function '_draw_children'
./lib/klove/kui.lua:727: in function 'draw'
./lib/klove/kui.lua:834: in function '_draw_children'
./lib/klove/kui.lua:727: in function 'draw'
./lib/klove/kui.lua:1059: in function 'draw'
./all-desktop/screen_map.lua:643: in function 'draw'
./all/director.lua:576 in function 'draw'
./main.lua:262: in function 'draw'
./main.lua:401" in function <./main.lua:349>
[C]: in function 'xpcall
```

**Comment by cigumo** [comment]
https://steamcommunity.com/app/816340/discussions/0/1735468693702927468/#c1751232561622418136
```
Video Card:
    Driver:  Intel Open Source Technology Center Mesa DRI Intel(R) Haswell Mobile x86/MMX/SSE2
    Driver Version:  3.0 Mesa 13.0.6
    OpenGL Version: 3.0
    Desktop Color Depth: 24 bits per pixel
```

**Comment by cigumo** [comment]
Fixed in 1.2.0
Added an ifdef so the same p_outline.c shader can be used for PC and Android. 
A definitive solution would be moving the default values initialization out of the shader and into shader_db, maybe with lua files in the shader dir.
